serde/src/test/org/apache/hadoop/hive/serde2/objectinspector/TestReflectionObjectInspectors.java

```diff
@@ -83,13 +85,57 @@ public void testReflectionObjectInspectors() throws Throwable {
       a.myMapStringString = new HashMap<String, String>();
       a.myMapStringString.put("key", "value");
 
-      assertEquals(1, soi.getStructFieldData(a, fields.get(0)));
-      assertEquals(2, soi.getStructFieldData(a, fields.get(1)));
-      assertEquals("test", soi.getStructFieldData(a, fields.get(2)));
-      assertEquals(a, soi.getStructFieldData(a, fields.get(3)));
-      assertEquals(a.myListString, soi.getStructFieldData(a, fields.get(4)));
+      int myIntIdx = IntStream.range(0, fields.size())
+        .filter(idx -> fields.get(idx)
+        .getFieldName()
+        .equals("myint"))
```

Comment:
All the names in `.equals()` here are lower case because `getFieldName()` will return field name in the lower case (i.e., it calls `.toLowerCase()` internally).